### ISSUE TITLE
Manual for users of assistive technology

### ISSUE THREAD

**Issue by rianrietveld** [WordPress/gutenberg#10373]
The new interface of Gutenberg has a learning curve, for all users.

For users that depend on assistive technology (AT), like a screen reader or voice recognition software, that learning curve is much higher.
If you don't have an overview of the whole screen or are depended on keyboard focus to perform tasks, Gutenberg is harder to learn.

To help AT users, a manual is needed. With:
* a walk through of the basic functionality with a keyboard only
* how to navigate through functionality with the screen reader navigation
* a list of accessible names for labels that are icons now
* a list of the keyboard short cuts
* a way to report issues or and contribute.

@abrightclearweb already started this and she need help, people who do research on how to use Gutenberg and people that help writing. The [work document is on Google Docs](https://docs.google.com/document/d/1pjYlOhkMu4NqcsNhKzJ_KTz2Z-yacMuX8UySslJp6oQ)

Related: #5420

**Comment by aaronjorbin** [comment]
This sounds great.

@abrightclearweb are you project managing this?  Are there specific subtasks that you can lay out that need to be done?

**Comment by abrightclearweb** [comment]
Hi @aaronjorbin - I started writing it, and @Wingo5315 also contributed.

I've been on holiday the last two weeks, so am intending to get back to it ASAP, with the 5.0 release due soon.

**Comment by chrisvanpatten** [comment]
Thanks for tackling this @Wingo5315 and @abrightclearweb ^__^

It would be awesome to include this in our user handbook — I updated my outline in #10911 to [reference this](https://github.com/WordPress/gutenberg/issues/10911#issuecomment-433012480).

Although I don't use any assistive technology myself, let me know if I can do anything to help your efforts and please feel free to join us in the weekly "Gutendocs" meeting on Slack (Tuesdays at 17:00 UTC in #docs) if you need any thing too 😄

**Comment by DaisyOlsen** [comment]
This issue seems best addressed in User Documentation.

**Comment by t-hamano** [comment]
Thank you for your suggestion. I think this is an improvement for the handbook, Learn WordPress content, etc., so I would like to close this issue.

If you have any suggestions for improvement in Gutenberg, please feel free to comment.
